src/specBuilder/bar/barTestUtils.ts

```diff
@@ -172,7 +170,6 @@ export const stackedLabelText = {
 	type: 'text',
 	from: { data: FILTERED_TABLE },
 	name: 'bar0_annotationText',
-	interactive: false,
```

Comment:
Do you know what code got added/modified that caused these to be removed?

Reply:
I'm not sure on this. I will restore the `interactive: false` line